shiny/README.md

```diff
@@ -0,0 +1,6 @@
+# Shiny Implementation
```

Comment:
Let's add provenance for the `ratio` files to the readme by including a stable link to their source.

Reply:
Addressed in 8246918 !